tests/setup_local_db/ddl_results_and_cdm.sql

```diff
@@ -99,4 +99,42 @@ CREATE VIEW omop.OBSERVATION_CONTINUOUS AS
 SELECT ob.person_id, ob.observation_concept_id, ob.value_as_string, ob.value_as_number, ob.value_as_concept_id
 FROM omop.observation ob
 INNER JOIN omop.concept concept ON concept.CONCEPT_ID=ob.OBSERVATION_CONCEPT_ID
-WHERE concept.CONCEPT_CLASS_ID='MVP Continuous' or concept.CONCEPT_ID=2000007027
+WHERE concept.CONCEPT_CLASS_ID='MVP Continuous' or concept.CONCEPT_ID=2000007027;
+
+CREATE VIEW omop.DATA_DICTIONARY AS
+WITH cte_counts AS (SELECT observation_concept_id,
+                           COUNT(DISTINCT person_id) AS number_of_people_with_variable,
+                           COUNT(DISTINCT CASE WHEN value_as_number IS NOT NULL THEN person_id END) AS number_of_people_where_value_is_filled_number,
+                           COUNT(DISTINCT CASE WHEN value_as_concept_id IS NOT NULL AND value_as_concept_id > 0 THEN person_id END) AS number_of_people_where_value_is_filled_concept,
+                           COUNT(DISTINCT CASE WHEN value_as_number IS NULL THEN person_id END) AS number_of_people_where_value_is_null_number,
+                           COUNT(DISTINCT CASE WHEN value_as_concept_id IS NULL OR value_as_concept_id = 0 THEN person_id END) AS number_of_people_where_value_is_null_concept
+                    FROM omop.OBSERVATION_CONTINUOUS
+                    GROUP BY observation_concept_id)
+SELECT c.vocabulary_id,
+       c.concept_id,
+       c.concept_code,
+       c.concept_name,
+       c.concept_class_id,
+       cc.number_of_people_with_variable,
+       CASE
+           WHEN c.concept_class_id = 'MVP Continuous' THEN cc.number_of_people_where_value_is_filled_number
+           ELSE cc.number_of_people_where_value_is_filled_concept END AS number_of_people_where_value_is_filled,
+       CASE
+           WHEN c.concept_class_id = 'MVP Continuous' THEN cc.number_of_people_where_value_is_null_number
+           ELSE cc.number_of_people_where_value_is_null_concept END  AS number_of_people_where_value_is_null,
+       CASE WHEN c.concept_class_id = 'MVP Continuous' THEN 'Number' ELSE 'Concept Id' END AS value_stored_as,
+       MIN(oc.value_as_number) AS min_value,
+       MAX(oc.value_as_number) AS max_value,
+       AVG(oc.value_as_number) AS mean_value,
+-- For sql server deployment, use stdev(value_as_number) instead of stddev(value_as_number)
```

Comment:
maybe add similar comments to the other parts for which we know there are differences (like count_big)?

Reply:
I think we do not need count_big since we cannot index the new view in either database right now.